gateway-ha/src/main/java/io/trino/gateway/ha/notifier/EmailNotifier.java

```diff
@@ -78,7 +77,7 @@ public void sendNotification(
                                 message.addRecipient(Message.RecipientType.TO, new InternetAddress(r));
                             }
                             catch (Exception e) {
-                                log.error("Recipient email [" + e + "] could not be added", e);
+                                log.error(e, "Recipient email [" + e + "] could not be added");
```

Comment:
Looks like the existing code is incorrect here. 
```
 log.error(e, "Recipient email [%s] could not be added", r);
```
makes sense to me, but as is this will print the exception in the middle of the log message